### PR TITLE
[logging] All logs with text sent to text logs

### PR DESCRIPTION
**Pull request by gregnazario**
### Overview
In order to allow us to debug live, all logs will now print the text portion to stdout.  In addition, renamed the `log` field to `text` to remove confusion.

This is currently going to have conflicts with my other two changes out to remove `send_struct_log!` and `crit!`, but I'll work them out as they go :)

I believe I will need to reformat the logs to be more useful with level and line information, will do as I test.

### Testing
TBD

### CODE REVIEW

File: common/logger/src/lib.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@
 //! use serde_json::Value;
 //!
 //! pub struct StructuredLogEntry {
-//!     log: Option<String>,
+//!     text: Option<String>,
```
Comment (bmwill) [inline common/logger/src/lib.rs:33]: If we're going about changing this, maybe `message` is a little more descriptive than text?

Comment (gregnazario) [reply common/logger/src/lib.rs:33]: Yeah, message is probably better.  Let me do some git magic to fix it up

File: common/logger/src/lib.rs [hunk 2]
```diff
@@ -292,15 +286,15 @@ macro_rules! location {
...
         $entry.add_pattern($fmt);
     };
     ($entry:ident, $fmt:expr,) => {
-        $entry.add_log(format!($fmt));
+        $entry.add_message(format!($fmt));
         $entry.add_pattern($fmt);
     };
     ($entry:ident, $fmt:expr, $($arg:tt)+) => {
-        $entry.add_log(format!($fmt, $($arg)+));
+        $entry.add_message(format!($fmt, $($arg)+));
```
Comment (davidiw) [inline common/logger/src/lib.rs:297]: can we get rid of these add_*? and just have *

**Comment by gregnazario** [comment]
> is this intentionally in draft?

It's still in draft, because I wanted to test it on a cluster test to ensure I can see text logs are printed out.  A lot had been changing on this PR

File: common/logger/src/lib.rs (hunk 2 above)
Comment (gregnazario) [reply common/logger/src/lib.rs:297]: Yeah, it's a little confusing because of the way the macros are built recursively, but it is possible.

**Comment by gregnazario** [comment]
Closing in favor of https://github.com/libra/libra/pull/5875

**Pull request closed without merging**